CDS_Dashboard/script.js

```diff
@@ -314,18 +294,39 @@ function clearModal() {
     });
 }
 
-var counter = 0;
+function buildOptions(drilldown, options) {
```

Comment:
I don't think you need to take `options` as an argument to this function. In both possible paths through the code, `options` gets totally redefined.